mmv1/products/networksecurity/ProjectAddressGroup.yaml

```
@@ -0,0 +1,65 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: "ProjectAddressGroup"
```

```suggestion
name: "ProjectAddressGroup"
legacy_name: "AddressGroup"
```

This should allow for the resource names to match in a more standard way -- i.e.
```
google_network_security_address_group_iam_policy - google_network_security_address_group
vs current 
google_network_security_project_address_group_iam_policy - google_network_security_address_group
```

---

Done